src/nonstandard.jl

```diff
@@ -56,3 +56,21 @@ constructorof(::Type{<:LinRange}) = linrange_constructor
 ### Expr: args get splatted
 # ::Expr annotation is to make it type-stable on Julia 1.3-
 constructorof(::Type{<:Expr}) = (head, args) -> Expr(head, args...)::Expr
+
+### Cholesky
+setproperties(C::LinearAlgebra.Cholesky, patch::NamedTuple{()}) = C
+function setproperties(C::LinearAlgebra.Cholesky, patch::NamedTuple{(:L,),Tuple{L}}) where {L<:LinearAlgebra.LowerTriangular}
+    return LinearAlgebra.Cholesky(C.uplo === 'U' ? copy(transpose(patch.L.data)) : patch.L.data, C.uplo, C.info)
```

Comment:
Maybe
```suggestion
    return LinearAlgebra.Cholesky(C.uplo === 'U' ? copy(patch.L.data') : patch.L.data, C.uplo, C.info)
```
to make it a bit more consistent with `getproperty(::Cholesky, ...)`?